checks/branch_protected.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 const branchProtectionStr = "Branch-Protection"
+const minReviews = 2
```

Comment:
This will likely cause more failures. IMO this will cause the projects to avoid this check. Most of the PR's don't require `2` up votes

Reply:
sg.